guides/common/modules/proc_troubleshooting-permission-issues.adoc

```diff
@@ -0,0 +1,26 @@
+[id="troubleshooting-permission-issues"]
+= Troubleshooting permission issues
+
+You can perform a health check on {ProjectServer} to find permission issues.
```

Comment:
Can you include a bit more context into the introduction? How will users know that they are in fact dealing with permission issues? Is there a specific action failure, an error message, etc.? Or something that  you expect users would try to look up with a full text search?

Reply:
I have slightly extended the introduction and referenced the redmine issue in my commit message. Should I include parts of the error message?

Reply:
I like the new introduction, thanks.

Regarding the error message: If the message is something you expect users might first see in their environment and then search for what it means, then yes, including it is a good idea. If you think this scenario is unlikely, then including the message is not necessary. Ultimately, your call :)